WalletWasabi.Fluent/Controls/TreeDataGrid.axaml

```diff
@@ -325,4 +327,70 @@
       </ControlTemplate>
     </Setter>
   </ControlTheme>
+
+  <!-- TreeDataGridRow: CoinjoinCoinsListView.axaml -->
+  <ControlTheme x:Key="CoinjoinCoinsListViewTreeDataGridExpanderCell" TargetType="TreeDataGridExpanderCell">
+
+    <Setter Property="Template">
+      <ControlTemplate>
+        <Border Background="{TemplateBinding Background}">
+          <DockPanel>
+            <Border DockPanel.Dock="Left"
+                    Margin="4 0 0 0"
+                    Width="12" Height="12">
+              <ToggleButton Theme="{StaticResource FluentTreeViewExpandCollapseChevron}"
+                            Focusable="False"
+                            IsChecked="{TemplateBinding IsExpanded, Mode=TwoWay}"
+                            Cursor="Hand">
+                <ToggleButton.IsVisible>
+                  <MultiBinding Converter="{x:Static BoolConverters.And}">
+                    <Binding Path="ShowExpander" RelativeSource="{RelativeSource TemplatedParent}" />
+                    <Binding Path="!IsChild" />
+                    <Binding Path="HasChildren" />
```

Comment:
For some reason that I don't understand adding the `DataContext` here breaks this